src/Nox.Types.EntityFramework/Types/DateTime/DateTimeDatabaseConfigurator.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+public class DateTimeDatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.DateTime;
+
+    public virtual bool IsDefault => true;
+
+    public void ConfigureEntityProperty(NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, EntityTypeBuilder builder, NoxSimpleTypeDefinition property, Entity entity, bool isKey)
+    {
+        builder
+            .OwnsOne(typeof(DateTime), property.Name, dtr =>
+            {
+                dtr.Property(nameof(DateTime.DateTimeValue))
+                    .HasConversion<DateTimeRangeConverter>();
```

Comment:
Hi @gjorgji-grgovski-authoritypartners , should we rename this dateTimeRangeConverter so it could be used both for datetime and datetimerange, or create a new converter